### PR TITLE
update npm/gem dependencies

### PR DESCRIPTION
**Pull request by devvmh**
needs ci + local testing

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,106 +1,530 @@
...
+      aws-sdk-core (~> 3)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-core (3.2.1)
+      aws-partitions (~> 1.0)
       aws-sigv4 (~> 1.0)
       jmespath (~> 1.0)
-    aws-sdk-resources (2.7.0)
-      aws-sdk-core (= 2.7.0)
-    aws-sigv4 (1.0.0)
+    aws-sdk-costandusagereportservice (1.0.0)
```
Comment (devvmh) [inline Gemfile.lock]: need to figure out if i can drop all these aws deps

File: package.json [hunk 2]
```diff
@@ -21,56 +21,56 @@
...
+    "babel-preset-react": "6.24.1",
     "backbone": "1.3.3",
-    "clipboard-js": "0.3.2",
-    "commonmark": "0.27.0",
-    "csv-parse": "1.1.10",
-    "emoji-mart": "0.3.7",
-    "getscreenmedia": "2.0.0",
+    "clipboard-js": "0.3.5",
+    "commonmark": "0.28.1",
+    "csv-parse": "1.2.1",
```
Comment (devvmh) [inline package.json]: test

File: package.json [hunk 3]
```diff
@@ -21,56 +21,56 @@
...
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
     "backbone": "1.3.3",
-    "clipboard-js": "0.3.2",
-    "commonmark": "0.27.0",
-    "csv-parse": "1.1.10",
-    "emoji-mart": "0.3.7",
-    "getscreenmedia": "2.0.0",
+    "clipboard-js": "0.3.5",
+    "commonmark": "0.28.1",
```
Comment (devvmh) [inline package.json]: test

File: package.json [hunk 4]
```diff
@@ -21,56 +21,56 @@
...
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
     "backbone": "1.3.3",
-    "clipboard-js": "0.3.2",
-    "commonmark": "0.27.0",
-    "csv-parse": "1.1.10",
-    "emoji-mart": "0.3.7",
-    "getscreenmedia": "2.0.0",
+    "clipboard-js": "0.3.5",
```
Comment (devvmh) [inline package.json]: test in multiple browers

File: package.json [hunk 5]
```diff
@@ -21,56 +21,56 @@
   "homepage": "https://github.com/metamaps/metamaps#readme",
   "dependencies": {
     "ajaxq": "0.0.7",
-    "attachmediastream": "1.4.2",
-    "autolinker": "1.4.0",
-    "babel-cli": "6.22.2",
-    "babel-loader": "6.2.10",
+    "attachmediastream": "2.0.0",
```
Comment (devvmh) [inline package.json]: erghhh test

File: package.json [hunk 6]
```diff
@@ -21,56 +21,56 @@
...
     "backbone": "1.3.3",
-    "clipboard-js": "0.3.2",
-    "commonmark": "0.27.0",
-    "csv-parse": "1.1.10",
-    "emoji-mart": "0.3.7",
-    "getscreenmedia": "2.0.0",
+    "clipboard-js": "0.3.5",
+    "commonmark": "0.28.1",
+    "csv-parse": "1.2.1",
+    "emoji-mart": "1.0.1",
```
Comment (devvmh) [inline package.json]: test

File: package.json [hunk 7]
```diff
@@ -21,56 +21,56 @@
...
-    "simplewebrtc": "2.2.2",
+    "react": "15.6.1",
+    "react-dom": "15.6.1",
+    "react-draggable": "3.0.3",
+    "react-dropzone": "4.1.2",
+    "react-onclickoutside": "6.5.0",
+    "react-router": "4.2.0",
+    "redux": "3.7.2",
+    "riek": "1.1.0",
+    "simplewebrtc": "3.0.1",
```
Comment (devvmh) [inline package.json]: test/revert i can't remember if this is safe to upgrade

File: package.json [hunk 8]
```diff
@@ -21,56 +21,56 @@
...
     "socket.io": "1.3.7",
-    "webpack": "2.2.1"
+    "webpack": "3.5.6"
```
Comment (devvmh) [inline package.json]: woahhhh test

File: package.json (hunk 2 above)
Comment (devvmh) [reply package.json]: works, but i do need one bugfix since csv import was broken overall

File: package.json (hunk 3 above)
Comment (devvmh) [reply package.json]: seems to work, without testing exhaustively

File: package.json (hunk 6 above)
Comment (devvmh) [reply package.json]: works

File: package.json (hunk 8 above)
Comment (devvmh) [reply package.json]: seems to work, and apparently it's an easy upgrade per https://medium.com/webpack/webpack-3-official-release-15fd2dd8f07b

File: package.json (hunk 4 above)
Comment (devvmh) [reply package.json]: meh

File: Gemfile.lock (hunk 1 above)
Comment (devvmh) [reply Gemfile.lock]: uhhh pinning aws-sdk to 2.7.0 since all these deps are dumb

File: package.json (hunk 5 above)
Comment (devvmh) [reply package.json]: video works

File: package.json (hunk 7 above)
Comment (devvmh) [reply package.json]: i guess video works.....?!?

i did have a small issue connecting on my first attempt. if this persists in prod after deploy i'll consider reverting

File: package.json [hunk 9]
```diff
@@ -21,56 +21,57 @@
...
-    "react-router": "3.0.2",
-    "redux": "3.6.0",
-    "riek": "1.0.7",
+    "prop-types": "15.5.10",
+    "react": "15.6.1",
+    "react-dom": "15.6.1",
+    "react-draggable": "3.0.3",
+    "react-dropzone": "4.1.2",
+    "react-onclickoutside": "6.5.0",
+    "react-router": "4.2.0",
```
Comment (Connoropolous) [inline package.json]: ah, this was a killer :p dropping it back to 3.0.5

**Pull request merged**